Sources/Sentry/SentryRequest.h

```diff
@@ -0,0 +1,50 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Request)
+@interface SentryRequest : NSObject <SentrySerializable, NSCopying>
```

Comment:
`m`: Any specific reason for implementing `NSCopying`? If not, you can ditch it.

Reply:
nope, copied from examples, removing it